templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -251,33 +256,73 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <files_dir path="/data/dp01/galaxy_db/files"/>
             <extra_dir type="temp" path="/data/jwd/object_store_temp/dataplant01"/>
             <extra_dir type="job_work" path="/data/jwd/main"/>
         </backend>
-
-        <!--
-        <backend type="generic_s3" id="s3_netapp01" allow_selection="false" weight="0" store_by="uuid" name="Short term storage for method development">
-            <description>S3 based objetct storage matained by the compute center of the University of Freiburg.
-                         This storage is purged weekly and so it is only appropriate for short term methods development and such.
-                         The rapid deletion of stored data enables us to provide this storage without a quota.
-                         More information about our storage offerings can be found [here](https://galaxyproject.org/eu/).
+        
+        <!-- also called Scratch storage -->
+        <backend type="generic_s3" id="s3_scratch_netapp01" allow_selection="true" weight="0" store_by="uuid" device="scratch" name="Short term storage for method development">
+            <description>S3-based object storage is maintained by the compute center of the University of Freiburg.
+                         Data on this storage, also called scratch storage, is deleted if it is older than two months (60 days), so it is only appropriate for short-term methods development.
+                         The rapid deletion of stored data enables us to provide this storage without a quota. This storage is not backed up.
+                         More information about our storage offerings is documented at [https://galaxyproject.org/eu/storage/](https://galaxyproject.org/eu/storage/).
             </description>
+            <quota enabled="false" source="scratch" />            
             <badges>
                 <less_stable />
                 <not_backed_up />
-                <short_term>The data stored here is purged after a week.</short_term>
+                <short_term>The data stored here is purged after two months (60 days).</short_term>
             </badges>
             <auth access_key="{{ s3_netapp01.usegalaxy_eu.access_key }}" secret_key="{{ s3_netapp01.usegalaxy_eu.secret_key }}" />
-            <bucket name="{{ s3_netapp01.usegalaxy_eu.bucket_name }}" use_reduced_redundancy="False" max_chunk_size="250" />
+            <bucket name="{{ s3_netapp01.usegalaxy_eu.bucket_name }}" use_reduced_redundancy="false" max_chunk_size="250" />
             <connection host="s3.bwsfs.uni-freiburg.de" port="443" is_secure="True" conn_path="" multipart="True"/>
-            <cache path="/data/jwd02f/s3_object_store_cache" size="10000" />
-            <extra_dir type="job_work" path="/data/jwd02f/main"/>
-            <extra_dir type="temp" path="/data/jwd02f/tmp"/>
+            <cache path="{{ object_store_cache_dir }}" size="10000" cache_updated_data="true" />
```

Comment:
Should we increase the cache size to a bit more than 10TiB? Likely to the size (or at least half of it) of the cache storage or something?